services/order.py

```diff
@@ -0,0 +1,41 @@
+from datetime import datetime
+from typing import List
+from django.db import transaction
+from typing import Optional
+
+from db.models import Order, MovieSession, User, Ticket
+
+
+def create_order(
+        tickets: List[dict],
+        username: str,
+        date: Optional[str] = None
+) -> Order:
+    with transaction.atomic():
+        if User.objects.filter(username=username):
+
+            new_order = Order.objects.create(
+                user=User.objects.get(username=username)
+            )

```

Comment:
Don't make two of the same request to db. You can store the result of `User.objects.filter(username=username)` and use it where you need.